tests/wpt/web-platform-tests/html/browsers/browsing-the-web/unloading-documents/prompt/004-1.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+004-1
+<script>
+onbeforeunload = function() {
+  onbeforeunload = null;
+  parent.beforeunload_fired = true;
+  setTimeout(function() {
+               parent.timeout_fired = true;
+             }, 1000);
+}
+onload = function() {
+  if (!parent.loaded) {
+    parent.loaded = true;
+    location="004-2.html?" + Math.random();
```

Comment:
Is there any reason for `Math.random()`?

Reply:
I think it's to prevent caching.